components/admin/admin-submission-status.tsx

```
@@ -16,36 +16,134 @@
+            <BarChart3 className="h-5 w-5 text-indigo-600 dark:text-indigo-400" />
+            Submission Status
+          </CardTitle>
         </CardHeader>
         <CardContent>
-          <div className="text-center text-gray-500 dark:text-gray-400 py-8">
-            No submissions data available
+          <div className="space-y-6">
+            {/* Demo Pie Chart */}
+            <div className="flex justify-center">
+              <div className="relative w-24 h-24 sm:w-32 sm:h-32">
+                <svg width="96" height="96" className="sm:w-32 sm:h-32 transform -rotate-90">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)